routes.js

```diff
@@ -115,11 +115,15 @@ const sharedRoutes = {
   },
   Issue: {
     screen: IssueScreen,
-    navigationOptions: ({ navigation }) => ({
-      title: `#${navigation.state.params.issue
-        ? navigation.state.params.issue.number
-        : 'Issue'}`,
-    }),
+    navigationOptions: ({ navigation }) => {
+      const issueNumberRegex = /issues\/([0-9]+)$/;
+      const { issue, issueURL, isPR } = navigation.state.params;
+      const number = issue ? issue.number : issueURL.match(issueNumberRegex)[1];
+
+      return {
+        title: isPR ? `Pull request #${number}` : `Issue #${number}`,
```

Comment:
I'd say `Pull Request` should be capitalized, but it depends on how it's done other places throughout the code.